test/functional/proxy_test.rb

```diff
@@ -0,0 +1,65 @@
+require File.join(File.dirname(__FILE__), 'test_helper')
+
+describe 'proxy' do
+  describe 'import-classes' do
+    let(:cmd) { ['proxy', 'import-classes'] }
+    let(:params) { ['--id=83'] }
+    let(:report) do
+      {
+        'message' => 'Successfully updated environment and puppetclasses from the on-disk puppet installation',
+        'environments_with_new_puppetclasses' => 2,
+        'environments_updated_puppetclasses' => 0,
+        'environments_obsolete' => 0,
+        'environments_ignored' => 0,
+        'results' => [{
+          'name' => 'development',
+          'actions' => [ 'new', 'update' ],
+          'new_puppetclasses' => [
+            'motd',
+            'hammer'
+          ],
+          'updated_puppetclasses' => [
+            'stdlib',
+            'vim'
+          ]
+        },{
+          'name' => 'production',
+          'actions' => [ 'obsolete', 'ignore' ],
+          'obsolete_puppetclasses' => [
+            'apache'
+          ],
+          'ignored_puppetclasses' => [
+            'hammer'
+          ]
+        }]
+      }
+    end
+
+    it 'prints detailed report' do
+      api_expects(:smart_proxies, :import_puppetclasses, 'Import classes')
+        .with_params('id' => '83')
+        .returns(report)
+
+      output = OutputMatcher.new([
+        'Result:               Successfully updated environment and puppetclasses from the on-disk puppet installation',
```

Comment:
Can we do something about the gap between `Result:` and the message? I probably understand where it comes from but here it doesn't make much visual sense. Any idea?
```
Result:               Successfully updated environment and puppetclasses from the on-disk puppet installation
Changed environments: 
 1) production
    New classes: 
     1) apt::backports
     2) apt
     3) apt::params
```
Can we change the numbering for an unnumbered list? In my testing imports I had more then hundred new classes and the numbering does not improve readability.